features/cookie.json

```diff
@@ -0,0 +1,110 @@
+{
+    "_meta": {
+        "description": "Restriction of 1st party cookies and 3rd party cookie blocking",
+        "sampleExcludeRecords": {
+            "domain": "example.com",
+            "reason": "site breakage"
+        }
+    },
+    "state": "enabled",
```

Comment:
Do we want to enable this for all? Or just override it in Chrome? Likely this will need to be enabled on Chrome and FF and `nonTrackerCookie` disabled on FF.

Reply:
"state" should be enabled for all and then the settings overriden in FF. I'll make the change; this was whilst testing :)

Reply:
Maybe just enabled for extensions? There's no use enabling it for native platforms and the config should be more clear in that regard.